src/components/bootcamp/layout/Footer.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react'
+import Container from '@mui/material/Container'
+import Typography from '@mui/material/Typography'
+import Link from '@mui/material/Link'
+
+interface CopyrightProps {
+  description: string
+}
+
+function Copyright(props: CopyrightProps) {
+  return (
+    <Typography
+      variant="body2"
+      color="text.secondary"
+      align="center"
+      style={{ marginBottom: '0.5%', color: 'white' }}>
+      <Link color="inherit" style={{ fontSize: '13px', textDecoration: 'none' }}>
+        © {new Date().getFullYear()} Pordkrepi.bg Bootcamp Module {props.description}
+      </Link>{' '}
+    </Typography>
+  )
+}
+
+interface FooterProps {
+  description: string
+  title: string
+}
+
+export default function Footer(props: FooterProps) {
+  const { description, title } = props
+
+  return (
+    <footer
+      style={{
+        position: 'fixed',
+        left: '0',
+        bottom: '0',
+        width: '100%',
+        backgroundColor: '#46dbf2',
+        textAlign: 'center',
+        color: 'white',
+        height: '6.8%',
+      }}>
```

Comment:
This style is inline and won't be cached during server side rendering.

You can do the following instead and move the styles to the container using `sx` property which is cached.

```tsx
<Container maxWidth="lg" component="footer" sx={...}>
```